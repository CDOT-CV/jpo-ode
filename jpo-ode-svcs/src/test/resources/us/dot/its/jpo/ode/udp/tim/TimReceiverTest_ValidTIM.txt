001f7b0a2020226d65746164617461223a207b0a202020202262736d536f75726365223a20224556222c0a20202020226c6f6746696c654e616d65223a202262736d54782e677a222c0a20202020227265636f726454797065223a202262736d5478222c0a20202020227365637572697479526573756c74436f6465223a202273756363657373222c0a202020202272656365697665644d65737361676544657461696c73223a207b0a202020202020226c6f636174696f6e44617461223a207b0a2020202020202020226c61746974756465223a202234302e35363537383831222c0a2020202020202020226c6f6e676974756465223a20222d3130352e30333136373432222c0a202020202020202022656c65766174696f6e223a202231343839222c0a2020202020202020227370656564223a2022302e34222c0a20202020202020202268656164696e67223a20223236372e34220a2020202020207d2c0a202020202020227278536f75726365223a20224e41220a202020207d2c0a20202020227061796c6f616454797065223a202275732e646f742e6974732e6a706f2e6f64652e6d6f64656c2e4f646542736d5061796c6f6164222c0a202020202273657269616c4964223a207b0a2020202020202273747265616d4964223a202238303137383063622d643931642d343434622d386634642d396334346665323766356561222c0a2020202020202262756e646c6553697a65223a2031362c0a2020202020202262756e646c654964223a2037322c0a202020202020227265636f72644964223a2031352c0a2020202020202273657269616c4e756d626572223a2031343734310a202020207d2c0a20202020226f646552656365697665644174223a2022323031392d30342d30395431383a30373a34372e3739385a222c0a2020202022736368656d6156657273696f6e223a20362c0a20202020227265636f726447656e6572617465644174223a2022323031382d30352d30315431353a35353a35352e3439345a222c0a20202020227265636f726447656e6572617465644279223a20224f4255222c0a202020202273616e6974697a6564223a2066616c73650a20207d2c0a2020227061796c6f6164223a207b0a20202020226461746154797065223a202275732e646f742e6974732e6a706f2e6f64652e706c7567696e2e6a323733352e4a3237333542736d222c0a202020202264617461223a207b0a20202020202022636f726544617461223a207b0a2020202020202020226d7367436e74223a203131392c0a2020202020202020226964223a20223331333235343333222c0a2020202020202020227365634d61726b223a2035353439342c0a202020202020202022706f736974696f6e223a207b0a20202020202020202020226c61746974756465223a2034302e353635373838312c0a20202020202020202020226c6f6e676974756465223a202d3130352e303331363734322c0a2020202020202020202022656c65766174696f6e223a20313438392e300a20202020202020207d2c0a202020202020202022616363656c536574223a207b0a2020202020202020202022616363656c4c6174223a20302e30302c0a2020202020202020202022616363656c4c6f6e67223a20302e34332c0a2020202020202020202022616363656c56657274223a20302e30302c0a2020202020202020202022616363656c596177223a20302e30300a20202020202020207d2c0a2020202020202020226163637572616379223a207b0a202020202020202020202273656d694d616a6f72223a20392e33302c0a202020202020202020202273656d694d696e6f72223a2031322e30350a20202020202020207d2c0a2020202020202020227472616e736d697373696f6e223a20224e45555452414c222c0a2020202020202020227370656564223a20302e34302c0a20202020202020202268656164696e67223a203236372e343030302c0a2020202020202020226272616b6573223a207b0a2020202020202020202022776865656c4272616b6573223a207b0a202020202020202020202020226c65667446726f6e74223a2066616c73652c0a20202020202020202020202022726967687446726f6e74223a2066616c73652c0a20202020202020202020202022756e617661696c61626c65223a20747275652c0a202020202020202020202020226c65667452656172223a2066616c73652c0a20202020202020202020202022726967687452656172223a2066616c73650a202020202020202020207d2c0a20202020202020202020227472616374696f6e223a2022756e617661696c61626c65222c0a2020202020202020202022616273223a2022756e617661696c61626c65222c0a2020202020202020202022736373223a2022756e617661696c61626c65222c0a20202020202020202020226272616b65426f6f7374223a2022756e617661696c61626c65222c0a20202020202020202020226175784272616b6573223a2022756e617661696c61626c65220a20202020202020207d2c0a20202020202020202273697a65223a207b0a20202020202020202020227769647468223a203139302c0a20202020202020202020226c656e677468223a203537300a20202020202020207d0a2020202020207d2c0a20202020202022706172744949223a205b0a20202020202020207b0a20202020202020202020226964223a202256656869636c65536166657479457874656e73696f6e73222c0a202020202020202020202276616c7565223a207b0a2020202020202020202020202270617468486973746f7279223a207b0a2020202020202020202020202020226372756d6244617461223a205b0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a20312e302c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303035312c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030303039372c0a2020202020202020202020202020202020202274696d654f6666736574223a20302e39300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a20342e372c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303134312c0a202020202020202020202020202020202020226c6f6e4f6666736574223a2033452d372c0a2020202020202020202020202020202020202274696d654f6666736574223a20332e36390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a20392e342c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303334312c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030303332362c0a2020202020202020202020202020202020202274696d654f6666736574223a20382e32390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a2031332e352c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303433382c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030303833392c0a2020202020202020202020202020202020202274696d654f6666736574223a2031382e32390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a2031322e302c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303438342c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313037322c0a2020202020202020202020202020202020202274696d654f6666736574223a2032362e34390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a2031312e372c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303437382c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313131372c0a2020202020202020202020202020202020202274696d654f6666736574223a2032382e30300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a2031312e332c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303332302c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313231312c0a2020202020202020202020202020202020202274696d654f6666736574223a2032382e38300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a2031312e322c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303330312c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313139372c0a2020202020202020202020202020202020202274696d654f6666736574223a2033302e30300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a20372e382c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303230322c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313039322c0a2020202020202020202020202020202020202274696d654f6666736574223a2033342e39300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a20312e392c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303233302c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313138362c0a2020202020202020202020202020202020202274696d654f6666736574223a2034302e30300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a202d302e312c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303234382c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313430312c0a2020202020202020202020202020202020202274696d654f6666736574223a2034322e38390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a202d332e332c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303334332c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313435312c0a2020202020202020202020202020202020202274696d654f6666736574223a2034392e38300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a202d332e362c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303434332c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313433372c0a2020202020202020202020202020202020202274696d654f6666736574223a2035312e36300a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a202d362e392c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303436362c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313431392c0a2020202020202020202020202020202020202274696d654f6666736574223a2035362e34390a202020202020202020202020202020207d2c0a202020202020202020202020202020207b0a20202020202020202020202020202020202022656c65766174696f6e4f6666736574223a202d382e342c0a202020202020202020202020202020202020226c61744f6666736574223a20302e303030303334362c0a202020202020202020202020202020202020226c6f6e4f6666736574223a20302e303030313236312c0a2020202020202020202020202020202020202274696d654f6666736574223a2036322e34390a202020202020202020202020202020207d0a20202020202020202020202020205d0a2020202020202020202020207d2c0a202020202020202020202020227061746850726564696374696f6e223a207b0a202020202020202020202020202022636f6e666964656e6365223a20302e302c0a2020202020202020202020202020227261646975734f664375727665223a20302e300a2020202020202020202020207d0a202020202020202020207d0a20202020202020207d2c0a20202020202020207b0a20202020202020202020226964223a2022537570706c656d656e74616c56656869636c65457874656e73696f6e73222c0a202020202020202020202276616c7565223a207b7d0a20202020202020207d0a2020202020205d0a202020207d0a20207d0a7d