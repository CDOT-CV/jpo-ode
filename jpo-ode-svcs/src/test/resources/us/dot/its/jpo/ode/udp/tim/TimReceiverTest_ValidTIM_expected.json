{
  "metadata" : {
    "securityResultCode" : "success",
    "recordGeneratedBy" : "RSU",
    "schemaVersion" : 9,
    "payloadType" : "us.dot.its.jpo.ode.model.OdeAsn1Payload",
    "serialId" : {
      "recordId" : 0,
      "serialNumber" : 0,
      "streamId" : "8006adcf-9fd7-447a-9d02-33005f703d08",
      "bundleSize" : 1,
      "bundleId" : 0
    },
    "sanitized" : false,
    "recordType" : "timMsg",
    "asn1" : "001F80A97014B901EC9C236B00000000000F775D9B0301EA73E452D1539716C99E9D555100003F0A59B080010007F8AA9979F4D3BB3A0A9266C000000854E3B2C47291F21E85EEF057980028422C1FFE0001FFFC00017FFF80000FFFFF000009FFFFF8000005FFFFFF00000007FF80007FFF00005FFFE00003FFFFC000027FFFFE0000017FFFFFC000000C523E43D138ECB11C6200C00FB0473DFB72A0E997C74007D408E5C376CD4F775D9B00",
    "maxDurationTime" : 0,
    "odeReceivedAt" : "2024-11-26T23:53:21.120Z",
    "originIp" : "127.0.0.1"
  },
  "payload" : {
    "data" : {
      "bytes" : "001F80A97014B901EC9C236B00000000000F775D9B0301EA73E452D1539716C99E9D555100003F0A59B080010007F8AA9979F4D3BB3A0A9266C000000854E3B2C47291F21E85EEF057980028422C1FFE0001FFFC00017FFF80000FFFFF000009FFFFF8000005FFFFFF00000007FF80007FFF00005FFFE00003FFFFC000027FFFFE0000017FFFFFC000000C523E43D138ECB11C6200C00FB0473DFB72A0E997C74007D408E5C376CD4F775D9B00"
    },
    "dataType" : "us.dot.its.jpo.ode.model.OdeHexByteArray"
  }
}